### ISSUE TITLE
filename extension .xxx

### ISSUE THREAD

**Issue by EmTee70** [szepeviktor/debian-server-tools#10]
Line 130 results in .xxx if not .txt if I read it right,
I changed it to 

```
if part.get_content_type() == 'text/plain':
    ext = '.txt'
elif part.get_content_type() == 'application/pdf':
    ext = '.pdf'
else:
    ext = '.xxx'
```
to ensure to get .pdf extension if it is a PDF, I know this is not originally from you, but I just found it here. Thank you.

**Comment by szepeviktor** [comment]
Hello @EmTee70 !

Thank you for your comment.
What file are you referring to?

This repo contains 1 zillion things.

**Comment by szepeviktor** [comment]
https://github.com/szepeviktor/debian-server-tools/blob/master/mail/extract-attachments/save_all_attachments.py#L171 ??

**Comment by EmTee70** [comment]
Yes, you are right. I am relative new to GitHub and so I was calling it an issue,  it would probably be cool to support all mime types. Put them in an associative array and so... maybe I try to solve this.

**Comment by szepeviktor** [comment]
No problem!

Just press the pencil icon (top right)
https://github.com/szepeviktor/debian-server-tools/blob/master/mail/extract-attachments/save_all_attachments.py
ads send your idea in a Pull Request!

**Comment by szepeviktor** [comment]
@EmTee70 Is it okay for me if you write your suggestion here as a snippet.
